content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -21,15 +19,16 @@ metrics that Mixer currently generates:
 
 To generate service-level metrics directly in the Envoy proxies, follow these steps:
 
-1.  To prevent duplicate telemetry generation, disable calls to `istio-telemetry` in the mesh:
-
+Option1:
```

Comment:
should there be a space between `option` and `1` (and likewise with `Option2` below)?

Reply:
ok done